libraries/Zigbee/src/ZigbeeEP.cpp

```diff
@@ -157,22 +175,32 @@ char *ZigbeeEP::readModel(uint8_t endpoint, uint16_t short_addr) {
   // clear read model
   _read_model = nullptr;
 
+  esp_zb_lock_acquire(portMAX_DELAY);
   esp_zb_zcl_read_attr_cmd_req(&read_req);
+  esp_zb_lock_release();
 
   //Wait for response or timeout
-  //Semaphore take
-  if (xSemaphoreTake(lock, portMAX_DELAY) != pdTRUE) {
+  if (xSemaphoreTake(lock, ZB_CMD_TIMEOUT) != pdTRUE) {
     log_e("Error while reading model");
   }
   return _read_model;
 }
 
-void ZigbeeEP::printBoundDevices() {
-  log_i("Bound devices:");
-  for ([[maybe_unused]]
-       const auto &device : _bound_devices) {
-    log_i("Device on endpoint %d, short address: 0x%x", device->endpoint, device->short_addr);
-    print_ieee_addr(device->ieee_addr);
+void ZigbeeEP::printBoundDevices(Print *print) {
+  if (print == nullptr) {
+    log_i("Bound devices:");
+    for ([[maybe_unused]]
+    const auto &device : _bound_devices) {
+      log_i("Device on endpoint %d, short address: 0x%x, ieee address: %02x:%02x:%02x:%02x:%02x:%02x:%02x:%02x", device->endpoint, device->short_addr, 
```

Comment:
```suggestion
      log_d("Device on endpoint %d, short address: 0x%x, ieee address: %02x:%02x:%02x:%02x:%02x:%02x:%02x:%02x", device->endpoint, device->short_addr, 
```

Reply:
Let's keep log_i for this scenario. If user calls printBoundDevices, and do not provide the Serial where to print, it will be printed with Log Info.